java-sdk/connectors/README.md

```diff
@@ -64,6 +64,35 @@ $base64AuthInfo = [Convert]::ToBase64String([Text.Encoding]::ASCII.GetBytes(("{0
 Invoke-RestMethod -Headers @{Authorization=("Basic {0}" -f $base64AuthInfo)} -Method PUT -ContentType "application/zip" "${FUSION_PROXY_URL}" -InFile "${PLUGIN_ZIP_PATH}"
 ```
 
+## Connect to Fusion Remotely
+
+Before trying to run plugins remotely, please go to [lucidworks](https://doc.lucidworks.com) to check how to configure fusion to run plugins remotely.
+
+A configuration file `config.yaml` holding the cluster information is needed in order to connect remotely to a fusion cluster.
+
+```bash
+pulsar:
+  service-url: <Pulsar Service URL>
+  admin-url: <Pulsar Admin URL>
+  tenant-name: <Pulsar Tenant Name (kube namespace)>
+  authenticationEnabled: <Pulsar authentication enabled flag>
+  tlsEnabled: <TLS enabled flag>
+proxy:
+  user: <Fusion proxy user>
+  password: <Fusion proxy password>
+  url: <Fusion proxy url>
+```
+NOTE: The `plugin.path` is not needed since the gradle task injects the plugin zip.
+
+The steps to use the `connect` tasks are the following:
+1. Create a `config.yaml` with your cluster information.
+2. Verify that the fusionVersion is correct on the gradle.properties file.
+3. Execute the `connect` task providing the plugin name and, the path to the configuration file.
+```bash
+./gradlew -p simple-connector connect -PconfigYamlPath=/path/to/config.yaml
+```
+4. After connecting to the cluster you should be able to see the plugin in Fusion's UI.
```

Comment:
should we reference the docs page for remote connectors here or it should be fine?